src/main/resources/mamute.properties

```diff
@@ -14,8 +14,6 @@ vraptor.errorcontrol.date.joda.pattern = dd/MM/yyyy
 tags.splitter.regex = \\,
 # Char used to split tags inside the JSP forms. It must matchs with the tags.splitter.regex property
 tags.splitter.char = ,
-# Regex of allowed characters for tag names
-tags.sanitizer.regex = [a-zA-Z0-9-]
```

Comment:
We could just use the Normalizer to url-encode those things, I guess it makes sense on every case we normalize a string, but maybe we should rename Normalizer to make this more explicit